bucket/w64devkit.json

```
@@ -0,0 +1,28 @@
+{
+    "version": "1.6.0",
+    "description": "Portable C and C++ Development Kit for x64 Windows",
+    "homepage": "https://github.com/skeeto/w64devkit",
+    "license": {
+        "identifier": " BSD-style",
+        "url": "https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/COPYING.MinGW-w64-runtime/COPYING.MinGW-w64-runtime.txt"
+    },
+    "architecture": {
+        "64bit": {
+            "url": [
+                "https://github.com/skeeto/w64devkit/releases/download/v1.6.0/w64devkit-1.6.0.zip"
+            ],
+            "hash": [
+                "3d02e902be730af348d2cda4971e78bea13ec7cca9c767eb3b3c26e37e6f3cad"
+            ]
+        }
+    },
+    "extract_dir": "w64devkit",
+    "env_add_path": "bin",
+    "checkver": {
+        "github": "https://github.com/skeeto/w64devkit/releases/latest",
+        "regex": "^(\\d+\\.)?(\\d+\\.)?(\\*|\\d+)$"
+    },
```

So instead of working checkver you had before, you proposed definitely non working and nonsense something?

Revert this nonsense and execute checkver script with force parameter...

```suggestion
    "checkver": "github",
```